man/rules/meson.build

```diff
@@ -1076,7 +1076,7 @@ manpages = [
  ['systemd-sysext',
   '8',
   ['systemd-confext', 'systemd-confext.service', 'systemd-sysext.service'],
-  ''],
+  'ENABLE_SYSEXT'],
```

Comment:
This meson file is autogenerated as the header at the top says, update the tag in manpage itself instead

Reply:
Whoops, sorry, didn't notice. Updated the man-page and regenerated the meson.build file.